### PR TITLE
fix: Change resource creation POST endpoint success status to 201

### LINKED ISSUE #156: Incorrect http status code for new resource creation POST endpoints
**Describe the bug**

For the POST endpoints in which new resource is being created like in `add_new_address` incorrect http status code i.e `200` is being shown.

**Expected behavior**

[201 Created](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) shown be shown instead

### PR DESCRIPTION
**Pull request by ashrafmahmood**
fixes #156

### CODE REVIEW

File: app/api/api_v1/endpoints/items.py [hunk 1]
```diff
@@ -23,20 +23,22 @@
...
     current_owner: Owner = Depends(get_current_owner),
+    response: Response,
 ):
     if get_shop_by_id(db, shop_id=shop_id, owner_id=current_owner.id):
         if item_by_name_and_shop(db, shop_id=shop_id, name=data.name):
             # Item name not unique within the shop. Show exception.
             raise HTTPException(
                 status_code=409, detail="Item with this name already exists."
             )
+        response.status_code = status.HTTP_201_CREATED
```
Comment (Haider8) [inline app/api/api_v1/endpoints/items.py]: Is this required now after you've added `status_code` in the `post` decorator?

Comment (ashrafmahmood) [reply app/api/api_v1/endpoints/items.py]: Yes, this is not required.

Comment (ashrafmahmood) [reply app/api/api_v1/endpoints/items.py]: @Haider8 Now check it.

**Comment by Haider8** [comment]
Please test each endpoint locally and change PR title to what you've done in this PR

**Comment by ashrafmahmood** [comment]
> Please test each endpoint locally and change PR title to what you've done in this PR

okay.

**Comment by ashrafmahmood** [comment]
> Please test each endpoint locally and change PR title to what you've done in this PR

I have tested each endpoint locally.

**Pull request merged**